platform/administer/templates/overview.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Flow, { Step } from "@site/src/components/Flow";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+import NavStep from "@site/src/components/NavStep";
+
+
+# Overview
+
+vCluster templates define reusable configurations for creating virtual Kubernetes clusters. They allow platform teams to standardize cluster creation while giving users a controlled way to customize certain settings. Templates are implemented using Kubernetes custom resources (`VirtualClusterTemplate`) and can be managed with tools such as `kubectl`, Helm, or ArgoCD.
+
+Each template defines the base configuration for a virtual cluster, including the Kubernetes version, control plane type, networking mode, storage settings, and sync behavior. Sync settings control which Kubernetes resources are synchronized between the virtual cluster and the host cluster—such as Service, Secret, ConfigMap, or custom resources. Templates can use sync rules to import host resources, export virtual cluster resources, or customize synchronization at the namespace or cluster scope.
```

Comment:
What is ‘control plane type’ and ‘network mode’ referring to?
Not sure I would use `Services as a sync example since they are synced by default - `Ingress` would be a good, common example.

Not sure I follow “Templates can use sync rules to import host resources, export virtual cluster resources, or customize synchronization at the namespace or cluster scope.” You can do this without a template with a regular vcluster.yaml, not sure it is necessary here as part of virtual cluster template documentation.